src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.hpp

```diff
@@ -0,0 +1,234 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <ostream>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Xcts::Solutions {
+
+/// Various coordinate systems in which to express the Schwarzschild solution
+enum class SchwarzschildCoordinates {
+  /*!
+   * \brief Isotropic Schwarzschild coordinates
+   *
+   * These arise from the canonical Schwarzschild coordinates by the radial
+   * transformation
+   *
+   * \f{equation}
+   * r = \bar{r}\left(1+\frac{M}{2\bar{r}}\right)^2
+   * \f}
+   *
+   * (Eq. (1.61) in \cite BaumgarteShapiro) where \f$r\f$ is the canonical
+   * Schwarzschild radius, also referred to as "areal" radius because it is
+   * defined such that spheres with constant \f$r\f$ have the area \f$4\pi
+   * r^2\f$, and \f$\bar{r}\f$ is the "isotropic" radius. In the isotropic
+   * radius the Schwarzschild spatial metric is conformally flat
+   *
+   * \f{equation}
+   * \gamma_{ij}=\psi^4\eta_{ij} \quad \text{with conformal factor} \quad
+   * \psi=\left(1+\frac{M}{2\bar{r}}\right)^4
```

Comment:
No ^4 here, since you're giving \psi, not \psi^4

Reply:
good catch :D